src/Microsoft.Health.Fhir.Shared.Api/Modules/SearchModule.cs

```diff
@@ -79,12 +80,7 @@ public void Load(IServiceCollection services)
             services.Add<SearchParameterStatusManager>()
                 .Singleton()
                 .AsSelf()
-                .ReplaceService<INotificationHandler<StorageInitializedNotification>>();
```

Comment:
>StorageInitializedNotification [](http://example.com/codeflow?start=53&length=30)

You probably want to delete this class as well.

Reply:
StorageInitializedNotification is still being published by the data layer, (cosmos or SQL) and handled by SearchParameterDefinitionManager